ldb/admin.py

```diff
@@ -60,10 +67,117 @@ def __init__(self, *args, **kwds):
+        errors = {}
+
+        for field in self.get_import_fields():
+            if str.startswith(field.attribute, "student__") or str.startswith(field.attribute, "member__"):
+                continue
+            if isinstance(field.widget, widgets.ManyToManyWidget):
+                continue
+            try:
+                self.import_field(field, obj, data)
+            except ValueError as e:
+                errors[field.attribute] = ValidationError(force_text(e), code="invalid")
+
+        # Student validation
+        obj.student.yearbook_permission = bool(data['student__yearbook_permission'])
+        obj.student.first_year = data['student__first_year']
+        obj.student.study = str(data['student__study'])
+        obj.student.student_number = str(data['student__student_number'])
+        obj.student.emergency_phone = str(data['student__emergency_phone'])
+        obj.student.emergency_name = str(data['student__emergency_name'])
+
+        try:
+            obj.student.full_clean(exclude='person')
+        except ValidationError as e:
+            for key, value in e:
+                if key is not 'person':
+                    errors["student__" + key] = ValidationError(force_text(value), code="invalid")
+
+        # Member validation
+        obj.member.amount_paid = data['member__amount_paid']
+        obj.member.date_from = datetime.datetime.today()
+
+        try:
+            obj.member.full_clean(exclude='person')
+        except ValidationError as e:
+            for key, value in e:
+                if key is not 'person':
+                    errors["member__" + key] = ValidationError(force_text(value), code="invalid")
+
+        if errors:
+            raise ValidationError(errors)
+
+    def after_save_instance(self, instance, using_transactions, dry_run):
+        # Saving student
+        instance.student.person_id = instance.id
+        instance.student.save()
+        # Saving member
+        instance.member.person_id = instance.id
+        instance.member.save()
+
+    class Meta:
+        model = Person
+        use_transactions = True
+        fields = ('id', 'initials', 'firstname', 'preposition', 'surname', 'netid', 'student__student_number', 'street_name', 'house_number', 'postcode', 'city', 'country', 'email', 'phone_mobile', 'gender', 'birthdate', 'student__yearbook_permission', 'mail_announcements', 'mail_company', 'mail_education', 'machazine', 'student__first_year', 'student__study', 'student__emergency_phone', 'student__emergency_name', 'member__amount_paid')
+        export_order = ('id', 'initials', 'firstname', 'preposition', 'surname', 'netid', 'student__student_number', 'street_name', 'house_number', 'postcode', 'city', 'country', 'email', 'phone_mobile', 'gender', 'birthdate', 'student__yearbook_permission', 'mail_announcements', 'mail_company', 'mail_education', 'machazine', 'student__first_year', 'student__study', 'student__emergency_phone', 'student__emergency_name', 'member__amount_paid')
```

Comment:
You could also reference `fields` here. (so `export_order = fields`). Not really necessary though.